src/relay/analysis/util.cc

```diff
@@ -454,5 +454,15 @@ bool IsDataDependant(const CallNode* call) {
 
   return tshape_data_dependant[op];
 }
+
+const Op& compiler_begin_op() {
```

Comment:
we probably don't want to put them under analysis namespace because they are used in the transform namespace. Can we just put them in the pass_util.h file and inline?

Reply:
Got it